server/handlers.go

```diff
@@ -716,7 +728,13 @@ func (s *Server) handleAuthCode(w http.ResponseWriter, r *http.Request, client s
 		return
 	}
 
-	accessToken := storage.NewID()
+	accessToken, err := s.newAccessToken(client.ID, authCode.Claims, authCode.Scopes, authCode.Nonce, authCode.ConnectorID)
```

Comment:
On line 556 you define the `accessToken` but here you use `:=` instead of `=`. Thus, when you set it in the callback query parameter on line 624 you are using the variable declared on line 556, not this one.  Thus, `access_token` is always empty in the callback.

Reply:
Thank you. This came in via one of the commits I built on top of, and I did not notice the error. Fixed in 5b66bf0.